scripts/data_ingestion/fetch_espn_mlb.py

```
@@ -0,0 +1,657 @@
+                counts['downloaded'] += result['downloaded']
+                counts['ingested'] += result['ingested']
+                counts['failed'] += result['failed']
+                print(f"Completed game {game_id}: {result}")
+            except Exception as e:
+                print(f"Error processing game {game_id}: {e}")
+                counts['failed'] += 1
+    
+    return counts
+
+
+def main():
```

Ruff (0.15.10)

[warning] Too many branches (27 > 12)

(PLR0912)